tests/test_ingress.py

```diff
@@ -20,38 +20,37 @@ def test_basic_ingress(self, kube_version):
 
         doc = docs[0]
 
-        annotations = jmespath.search("metadata.annotations", doc)
-        assert len(annotations) > 1
-        assert annotations["kubernetes.io/ingress.class"] == "RELEASE-NAME-nginx"
+        assert doc["kind"] == "Ingress"
 
-        _, minor, _ = (int(x) for x in kube_version.split("."))
+        assert len(doc["metadata"]["annotations"]) >= 4
+        assert (
+            doc["metadata"]["annotations"]["kubernetes.io/ingress.class"]
+            == "RELEASE-NAME-nginx"
+        )
 
-        if minor >= 19:
-            assert doc["apiVersion"] == "networking.k8s.io/v1"
-            assert "RELEASE-NAME-astro-ui" in [
-                name[0]
-                for name in jmespath.search(
-                    "spec.rules[*].http.paths[*].backend.service.name", doc
-                )
-            ]
-            assert "astro-ui-http" in [
-                port[0]
-                for port in jmespath.search(
-                    "spec.rules[*].http.paths[*].backend.service.port.name", doc
-                )
-            ]
+        # This would be valid python, but we laod from json just to keep linters happy and the data more compact
+        expected_rules_v1beta1 = json.loads(
```

Comment:
@andriisoldatenko what do you think about this? There's a large data structure, and `black` wanted to break it up, and `flake8` was complaining about it being so long and stuff, so I am loading it from `json`. Seem OK?

Reply:
yeah json looks okey for expected results, easy to read than big data structures in python.